docs/progenitor-client.md

```diff
@@ -62,23 +62,27 @@ It can be used as the type `T` in most instances and extracted as a `T` using
 
 ## `Error<E>`
 
-There are four sub-categories of error covered by the error type variants:
+There are five sub-categories of error covered by the error type variants:
+
+- A request that did not conform to API requirements.
```

Comment:
```suggestion
- A request that did not conform to API requirements. This can occur when required builder or body parameters were not specified, and the error message will denote the specific failure.
```